content/projects/data-science-specific/data-wrangling/_index.md

```diff
@@ -79,25 +80,24 @@ df.apply(get_percentage, axis = 1) #axis=1 applies the function to all columns
+3. Creat a function (or functions) to support creating new columns containing the total score of each of the personality test subscales. The new columns should be named as per the **big five personality traits** laid out in [scoring](scoring.txt). In other words, for the **conscientiousness** total score, all items marked as belonging to 'conscientiousness' should be summed.
 
-The new data frame will look something like this and it should be named `personality_score_df`:
+The new data frame will look something like this and it should be named `personality_score_totals_df`:
 
-| I am always prepared | I am easily disturbed | I am exacting (demanding) in my work | ... | Conscientiousness | Emotional_Stability |
+| I am always prepared | I am easily disturbed | I am exacting (demanding) in my work | ... | Conscientiousness | Neuroticism |
 | -------------------- | --------------------- | ------------------------------------ | --- | ----------------- | ------------------- |
 | (3, 5)               | (4, 5)                | (3, 5)                               | ... | 10                | 5                   |
 | (3, 5)               | (4, 1)                | (3, 1)                               | ... | 6                 | 1                   |
 | (3, 5)               | (4, 3)                | (3, 3)                               | ... | 8                 | 3                   |
 
-3. Import the data in [departments.csv](departments.csv). This DataFrame should be named `department_df` Merge this DataFrame with the personality score DataFrame, keeping all applicants within the department DataFrame. Use an assert statement to check that the newly created merged DataFrame has the same amount of rows as the department DataFrame and the expected number of columns. The merged DataFrame should be named `merged_personality_department_df`
+4. Import the data in [departments.csv](departments.csv) and store it in a DataFrame. Merge this DataFrame with the personality score totals DataFrame. Again, use assertion techniques from [assertive programing tricks](projects/data-science-specific/assertive-programming-tricks-for-pandas/_index.md) to check that the newly created merged DataFrame has the expected number of rows and columns. The merged DataFrame should be named `merged_personality_department_df`
```

Comment:
On what column should we merge? Will it be obvious? Is there a difference between left, right, inner and outer joins here and is it worth mentioning at all?

Reply:
It's obvious - department DF only has ID and department